packages/page-staking/src/Overview/CurrentList.tsx

```diff
@@ -5,11 +5,9 @@
 import { DeriveHeartbeats, DeriveStakingOverview } from '@polkadot/api-derive/types';
 import { AccountId } from '@polkadot/types/interfaces';
 import { AddressDetails } from './types';
-
```

Comment:
Please revert these line changes. Here and elsewhere. There is always a split between -

- types
- external imports
- local imports